<ui version="4.0">
 <class>TGo4CommandLine</class>
 <widget class="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>43</height>
   </rect>
  </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
  <property name="windowTitle">
   <string>GUI Command line</string>
  </property>
  <property name="windowIcon">
    <iconset>:/icons/go4logo2_big.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="GUILabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GUI command:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGo4CommandsHistory" name="InputLine" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Type root or go4 command here (&quot;go4-&gt;&quot; to access go4 command interface functions). Return key will start execution</string>
     </property>
     <property name="maxCount" stdset="0">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ExecuteBut">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Execute command line.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <pixmap>:/icons/OK16.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="PredefinedBut">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select predefined macro statement. Then drag and drop histograms from browser to empty string arguments. Edit other arguments in command line.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>:/icons/macro_t.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="FileSearchBut">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Browse for macro to execute</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>:/icons/findfile.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
  <customwidgets>
    <customwidget>
      <class>QGo4CommandsHistory</class>
      <extends>QComboBox</extends>
      <header>QGo4CommandsHistory.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
</ui>
